### PR TITLE
Sound effects

### LINKED ISSUE #5: Add sounds to the game

### PR DESCRIPTION
**Pull request by wildjames**

### CODE REVIEW

**Comment by wildjames** [comment]
This still needs work. There's a bug where the death sound blocks the actual player death.

**Comment by Jibbajabbafic** [comment]
You can leave the PR open and set it to a draft PR instead to show it's still in progress

**Comment by wildjames** [comment]
Should be ok to merge in now. Had to slightly tweak the player class.

File: scripts/controllers/Lobby.gd [hunk 1]
```diff
@@ -21,15 +21,11 @@ remote func player_joined():
...
-func _on_Start_pressed():
-	rpc("begin_game")
-
 
-func _on_UpdateIPLabels_timeout():
-	# Update IPs
-	$Connecting/MyIP.text = "IP: " + str(Net.public_ip)
-	$Host/MyIP.text = "IP: " + str(Net.public_ip)
+func _on_BGMusic_finished():
+	pass #$BGMusic.play()
```
Comment (Jibbajabbafic) [inline scripts/controllers/Lobby.gd:40]: No repeating music? :'(

**Review by Jibbajabbafic — DISMISSED** [review]
Hahaha holy shit these are hilarious, I love it. Sadly WAV files are huge though so could you convert all the sounds to .ogg instead please?

**Comment by Jibbajabbafic** [comment]
Alternatively you could put the original WAVs in a 'raw_assets' folder and then stick the compressed .oggs into the audio folder

File: scripts/controllers/Lobby.gd (hunk 1 above)
Comment (wildjames) [reply scripts/controllers/Lobby.gd:40]: I know how much you love the sound of my smacking lips

**Review by wildjames — COMMENTED** [review]
Moved to OGG. Kept original WAV in a `Raw` subdirectory.

I think this should be ready for merge, and compatible with some p2p multiplayer changes I've made. However, there are some multiplayer bugs present in this branch that are fixed on my Multiplayer one. Should I merge those changes into here, or open a second PR?

**Comment by Jibbajabbafic** [comment]
Feel free to make a new PR

**Review by Jibbajabbafic — APPROVED** [review]

**Pull request merged**